### PR TITLE
trying to spruce up our native experience

### LINKED ISSUE #138: electron professional stuff
- [ ] correct icons
- [ ] name the app

### PR DESCRIPTION
**Pull request by jolesbi**
- changed some config language
- added some icons

closes #138

### CODE REVIEW

**Comment by nylira** [comment]
I think the scaling got out of whack for the logo in Sketch because the outlines are too thin. Here's how it should look in large sizes:

[black.pdf](https://github.com/cosmos/cosmos-ui/files/1520266/black.pdf)
black

**Comment by jolesbi** [comment]
good eye @nylira — i used an icon generator — will fix.

File: app/src/main/index.js [hunk 1]
```diff
@@ -90,15 +91,22 @@ function shutdown () {
...
   mainWindow = new BrowserWindow({
     minWidth: 320,
     minHeight: 480,
     width: 1200,
     height: 800,
+    center: true,
+    title: 'Cosmos',
+    icon: appIcon,
+    darkTheme: true,
+    titleBarStyle: 'hiddenInset',
```
Comment (jolesbi) [inline app/src/main/index.js]: @nylira this removes the grey bar at the top — but leaves the 3 buttons overlapping the logo...

File: app/src/main/index.js [hunk 2]
```diff
@@ -90,15 +91,22 @@ function shutdown () {
...
     webPreferences: { webSecurity: false }
   })
-  // mainWindow.maximize()
+  mainWindow.maximize()
```
Comment (jolesbi) [inline app/src/main/index.js]: i think this is a nice to have

Comment (nylira) [reply app/src/main/index.js]: Yes, agreed!

File: app/src/main/index.js (hunk 1 above)
Comment (nylira) [reply app/src/main/index.js]: I *love* how this looks. There are some issues with this though:

1. The app menu on mobile is also blocked by the 3 buttons. We can resolve this by moving the app menu to the right hand side.
2. There's no longer any way to drag the app window around. @jolesbi can you investigate how to make the top 48px of our app draggable?
3. How does this look on Windows?

Comment (faboweb) [reply app/src/main/index.js]: Please add screenshots of what to compare to. ;) I can't see any change:
image

**Comment by jolesbi** [comment]
@nylira yep, i can figure out how to make the app draggable still. and we will have to make some adjustments for the top area.

**Comment by jolesbi** [comment]
screen shot 2017-12-01 at 5 47 27 pm

**Comment by jolesbi** [comment]
@nylira i added the draggability in! we should talk about sidebar alignment stuff... you'll see when you open the app!

**Comment by jolesbi** [comment]
closes #164 too!

**Comment by nylira** [comment]
Okay this looks awesome:

screen shot 2017-12-05 at 11 26 42 am


With your other PR @jolesbi on increasing the height of the PageHeader, it's going to appear well balanced.

**Comment by faboweb** [comment]
Moving the maximize window line to another PR to merge this. I am opposed to this, as this feels very unnatural. There we can continue discussing it.

**Pull request merged**